### DISCUSSION TITLE
Holiday Schedule

### DISCUSSION THREAD

**Discussion by dandroid213** [chadwpalm/PrerollPlus#10] [Ideas]
Happy Thanksgiving everyone! And the holiday schedule had me thinking. What about a specific holiday setting for those that don't have set days every year? Just like today's! Might be good for other countries as well like India with Diwali and Chinese New Year.

**Comment by chadwpalm** [comment]
This is a good suggestion! I know when I was setting up some of my holidays over the course of the year I realized that I would have to keep some of them updated since they don't always fall on the same date.

Currently, when setting up sequences, it checks your schedule against the schedules of all of the other sequences and will not let you save the sequence if there is a conflict. One concern could be that if the holiday is floating around from year to year that it might jump in our out of another existing sequence causing a conflict. However, I could resolve the conflict by always making the sequence with a holiday take precedence over any other manually scheduled sequences for that day.

I'm open to additional thoughts about this.

**Comment by chadwpalm** [comment]
I've updated to v1.1.0 which now includes setting specific holidays. It's currently available in the develop branch/tag.
